polaris-react/src/components/DataTable/README.md

```diff
@@ -583,12 +583,12 @@ function DataTableFixedFirstColumnExample() {
 }
 ```
 
-### Data table with increased density and zebra striping
+### Data table with increased density
```

Comment:
I separated out zebra striping and increased density since they don't have anything in common in the Polaris context. It's in line with the other examples, where each example corresponds to one feature.